src/main/configService.ts

```diff
@@ -6,15 +6,24 @@ import { EventEmitter } from "stream";
 type ConfigurationSchema = {
     storagePath: string,
     bufferStoragePath?: string,
-    logPath: string,
-    logPathClassic?: string,
+    retailLogPath?: string,
+    classicLogPath?: string,
     maxStorage: number,
     monitorIndex: number,
     selectedCategory: number,
     audioInputDevice?: string,
     audioOutputDevice?: string,
     minEncounterDuration: number,
     startUp?: boolean,
+    recordRetail: boolean,
+    recordClassic: boolean,
+    recordRaids: boolean,
+    recordDungeons: boolean,
+    recordTwoVTwo: boolean,
+    recordThreeVThree: boolean,
+    recordSkirmish: boolean,
+    recordSoloShuffle: boolean,
+    recordBattlegrounds: boolean,
```

Comment:
I'm thinking we make this an array like `recordCategories` which just has the labels like "2v2", "Mythic+" but it's not super important.